cmd/plugins/juju-metadata/imagemetadata.go

```diff
@@ -26,12 +27,12 @@ import (
 	"github.com/juju/juju/version"
 )
 
-func prepare(context *cmd.Context, controllerName string, store jujuclient.ClientStore) (environs.Environ, error) {
+func prepare(ctx *cmd.Context, controllerName string, store jujuclient.ClientStore) (environs.Environ, error) {
```

Comment:
We should investigate how much effort it would require to thread a context through `cmd.Context` so we can pass it through.

Reply:
The BootstrapContext also has a context.Context field.